.changeset/proud-jeans-bow.md

```diff
@@ -0,0 +1,5 @@
+---
+"@itwin/presentation-hierarchies-react": patch
+---
+
+Adjust how `useUnifiedSelectionTree` determines `isReloading` value. `IsReloading` is now set to `true` if root nodes are being loaded.
```

Comment:
This change updated behavior for all `useTree` hook variants so maybe something like this would be better?

```suggestion
Adjust how `useTree` and other variants of it determines `isReloading` value. `isReloading` is now set to `true` if root nodes are being loaded.
```